gabbi/httpclient.py

```diff
@@ -172,15 +172,10 @@ def _verbose_output(self, message, prefix='', color=None, stream=None):
 
 def get_http(verbose=False, caption=''):
     """Return an Http class for making requests."""
```

Comment:
`s/Http/HTTP/`?

Reply:
No, the class is named `Http`. It does `HTTP`.

Reply:
I've now added d47f9430f96e8be044d2097005186213c3a7d275 - feel free to omit that, as I'm not sure it's entirely correct.